homeassistant/components/stream/recorder.py

```diff
@@ -122,7 +122,7 @@ def write_segment(segment: Segment) -> None:
             if not output_v:
                 output_v = output.add_stream(template=source_v)
                 context = output_v.codec_context
-                context.flags |= "GLOBAL_HEADER"
+                context.global_header = True
```

Comment:
Both options work. `global_header` is a property - its setter modifies `flags`.
In `13.1.0` only `global_header` is exposed in the `pyi` file (probably an overside but nevertheless).